README.md

```diff
@@ -2,9 +2,25 @@
 
 Website to find a court or tribunal.
 
-## Getting Started..
-
-### Prerequisites.
+## Table of Contents
+
+* [Getting Started](#setting-started)
+  * [Prerequisites](#prerequisites)
+  * [Running the application](#running-the-application)
+  * [Running with Docker](#running-with-docker)
+* [Developing](#developing)
+  * [Code style](#code-style)
+  * [Updating SCSS and JavaScript](#updating-scss-and-javascript)
+  * [Running the tests](#running-the-tests)
+  * [Security](#security)
+    * [CSRF prevention](#csrf-prevention)
+    * [Helmet](#helmet)
+  * [Healthcheck](#healthcheck)
+* [License](#license)
+
+## Getting Started
```

Comment:
This section looks empty

Reply:
I think that was just the title section for the 3 subsections below it. The bits to get started are in the subsections i think.